loader/gpa_helper.c

```diff
@@ -0,0 +1,254 @@
+/*
+ *
+ * Copyright (c) 2015-2021 The Khronos Group Inc.
+ * Copyright (c) 2015-2021 Valve Corporation
+ * Copyright (c) 2015-2021 LunarG, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Author: Jon Ashburn <jon@lunarg.com>
+ * Author: Charles Giessen <charles@lunarg.com>
+ */
+
+#include "gpa_helper.h"
+
+#include <string.h>
+
+#include "loader.h"
+#include "debug_utils.h"
+#include "wsi.h"
+
+void *trampolineGetProcAddr(struct loader_instance *inst, const char *funcName) {
```

Comment:
Should you change this to snake case?

Reply:
Probably! I focused on the functions in `loader.c` for the snake_case change, didn't try to rename everything under the sun.